### ISSUE TITLE
Brand Image example is missing critical alt text [docs]

### ISSUE THREAD

**Issue by andypillip** [twbs/bootstrap#36770]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The [example code for the navbar brand image](https://getbootstrap.com/docs/5.1/components/navbar/#image) fails to provide text for the home link due to the empty `alt` attribute.

```html
<nav class="navbar navbar-light bg-light">
  <div class="container">
    <a class="navbar-brand" href="#">
      <img src="/docs/5.1/assets/brand/bootstrap-logo.svg" alt="" width="30" height="24">
    </a>
  </div>
</nav>
```

Of course, it is not possible to provide an alt text that fits the user’s logo, but the example should give a good example, and at least the text along with it should mention that the company’s name should be put in the alt text.

How about “B for Bootstrap” in this example? Or simply “Bootstrap”

See [WebAIM’s recommendation on logo alt texts](https://webaim.org/techniques/alttext/#logos)

### Reduced test cases

N/A

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

v5.1

**Comment by andypillip** [comment]
Thank you very much for this rapid fix.

But the second case with an empty alt was actually ok. The text “Bootstrap” contained in the logo is written right next to it, so empty alt would be the best solution. “Logo” is not recommended in any case, see the linked recommendations for logo alt texts
